### ISSUE TITLE
Multi User Chest has an issue recovering tomb stones

### ISSUE THREAD

**Issue by FelixReuthlinger** [MSchmoecker/No-Chest-Block#6]
In multi player I often encountered that getting your equipment from a tombstone is buggy when using MultiUserChest (together with Epic Loot?). I also posted a but to EL discord, but I guess Randy does not work on it anymore again...
I really had to literally remove multi user chest mod, then the bug was gone. Otherwise the other player in the area of the tomb stone had to leave the area, so the owner of the tomb stone could open it again. The behavior was that when pressing E key to open, nothing happened. But an exception was thrown:
```
[Error  : Unity Log] InvalidOperationException: Collection was modified; enumeration operation may not execute.
Stack trace:
System.Collections.Generic.Dictionary`2+KeyCollection+Enumerator[TKey,TValue].MoveNext () (at <695d1cc93cca45069c528c15c9fdd749>:0)
EpicLoot.Data.ItemInfo.LoadAll () (at <3445a66d20694f088e28b9e0c6e99344>:0)
EpicLoot.Data.ItemInfo.IsStackableWithOtherInfo (EpicLoot.Data.ItemInfo info) (at <3445a66d20694f088e28b9e0c6e99344>:0)
EpicLoot.Data.ItemInfo.IsStackable (ItemDrop+ItemData a, ItemDrop+ItemData b) (at <3445a66d20694f088e28b9e0c6e99344>:0)
EpicLoot.Data.ItemInfo.CheckItemDataStackableAddItem (Inventory __instance, ItemDrop+ItemData item, System.Int32 x, System.Int32 y, System.Collections.Generic.Dictionary`2[System.String,System.String]& __state, System.Boolean& __result) (at <3445a66d20694f088e28b9e0c6e99344>:0)
(wrapper dynamic-method) Inventory.DMD<Inventory::AddItem>(Inventory,ItemDrop/ItemData,int,int,int)
Inventory.MoveAll (Inventory fromInventory) (at <0a1a436be26b421bb46752e573a7aa2f>:0)
MultiUserChest.InventoryHelper.MoveAll (Inventory from, Inventory to) (at D:/Projects/No-Chest-Block/MultiUserChest/Helper/InventoryHelper.cs:145)
MultiUserChest.InventoryHelper.GetAllMoveableItems (Inventory from, Inventory to, MultiUserChest.InventoryHelper+MoveAction moveAction) (at D:/Projects/No-Chest-Block/MultiUserChest/Helper/InventoryHelper.cs:135)
MultiUserChest.InventoryHelper.GetAllMoveableItems (Inventory from, Inventory to) (at D:/Projects/No-Chest-Block/MultiUserChest/Helper/InventoryHelper.cs:141)
MultiUserChest.ContainerHandler.TakeAll (Container container, Inventory playerInventory) (at D:/Projects/No-Chest-Block/MultiUserChest/ContainerHandler.cs:10)
MultiUserChest.Patches.ContainerPatch.TakeAllPatch (System.Boolean& __result, Container __instance, Humanoid character) (at D:/Projects/No-Chest-Block/MultiUserChest/Patches/ContainerPatch.cs:103)
(wrapper dynamic-method) Container.DMD<Container::TakeAll>(Container,Humanoid)
(wrapper dynamic-method) TombStone.DMD<TombStone::Interact>(TombStone,Humanoid,bool,bool)
(wrapper dynamic-method) Player.DMD<Player::Interact>(Player,UnityEngine.GameObject,bool,bool)
(wrapper dynamic-method) Player.DMD<Player::Update>(Player)
```

Maybe you can have a look if there is something you can do on this side? I would really love to make use of the mod again.

Maybe tomb stones could be excluded from any logic that MUC does add to the game?

**Comment by MSchmoecker** [comment]
This is a bug in EpicLoot duo to an outdated DataManager implementation. I will try to contact Randy and Vapok again about this, it's a known issue for quite some time now.
Otherwise I don't want to disable features of my mod because another mod throws errors, at most the tomb stones could be ignored if EpicLoot is installed.

**Comment by MSchmoecker** [comment]
Seems to be fixed in EpicLoot 0.9.9
